extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -376,7 +376,7 @@ export class ProjectsController {
 		}
 	}
 
-	public async exclude(context: FileNode | FolderNode): Promise<void> {
+	public async exclude(context: BaseProjectTreeItem): Promise<void> {
```

Comment:
I am sure it does but just to double confirm - It continues to work in current view and works in new view too right?

Reply:
after this change, the actions won't work in the old explorer view, but it'll still work in main since this is going into the separate branch. I'm going to remove the old view in the file explorer in this branch soon.